apps/crn-server/src/entities/event.ts

```diff
@@ -101,7 +89,7 @@ export const parseGraphQLSpeakers = (
         ?.filter((t) => t.id && t.id[0]?.id === team.id)
         .filter((s) => s.role)[0]?.role || undefined;
 
-    if (!user || !role) {
+    if (!user || !role || user.flatData.onboarded !== true) {
```

Comment:
when there is only a team not the user among the speakers data, I saw that appears this message `Speaker was not announced`
<img width="1408" alt="Screen Shot 2022-09-22 at 11 21 28" src="https://user-images.githubusercontent.com/16595804/191772796-210d0b4b-73e1-4b62-9b9a-54eb8c5eb33e.png">

Do we want to leave it like that? :)

Reply:
i think we probably do, it's pretty much the same behaviour as if there was no user. What we are saying is that attaching a speak who isn't onboarded (read - not active) is the same as attaching no speakers

Reply:
Ok! :) just wanted to confirm that 😉

Reply:
but i think i might make a note of that in the ticket as it is a very good point